app/javascript/Search/SearchForm.jsx

```diff
@@ -40,11 +28,8 @@ export const SearchForm = forwardRef(({ searchTerm, onSubmitSearch }, ref) => (
         />
         <Button
           type="submit"
-          variant="ghost"
-          contentType="icon-rounded"
           icon={SearchIcon}
```

Comment:
(small blocking thing) I'm seeing a prop-types error for the icon:

```
base.js:7677 Failed prop type: n[e] is not a function
  in ButtonNew
  in ForwardRef()
  in Search
  in SearchFormSync
```

I think the issue is the `PropTypes.ReactNode` in the Button component same as you found in the `Icon` component. If we also change the `Button` prop type to ` icon: PropTypes.elementType,` then the error goes away

Reply:
Oh right, that could be it...